cmake/nanobind-config.cmake

```diff
@@ -194,6 +190,7 @@ function (nanobind_build_library TARGET_NAME)
     ${NB_DIR}/src/nb_static_property.cpp
     ${NB_DIR}/src/nb_ft.h
     ${NB_DIR}/src/nb_ft.cpp
+    ${NB_DIR}/src/nb_foreign.cpp
```

Comment:
If this feature is officially called out as "metabind" or "pymetabind", should the file be named `nb_metabind.cpp` (for the nanobind specific metabind integration)